src/ucp/tag/offload.c

```diff
@@ -272,7 +272,9 @@ ucp_tag_offload_do_post(ucp_request_t *req)
                                         req->recv.length, req->recv.datatype,
                                         &req->recv.state, req->recv.mem_type,
                                         req, UCT_MD_MEM_FLAG_HIDE_ERRORS);
-        if (status != UCS_OK) {
+        if ((status != UCS_OK) || !req->recv.state.dt.contig.md_map) {
+            /* Can't register this buffer on the offload iface */
```

Comment:
`/* ... on the offload MD */` ?

Reply:
imo, `iface` is also ok, because we post receives to the ifaces, so need to register on its MD
will change if there will be other comments

Reply:
> imo, `iface` is also ok, because we post receives to the ifaces, so need to register on its MD
> will change if there will be other comments

yes, I agree. both are OK